packages/Adapters/POD_C/src/DTK_NoName.h

```diff
@@ -0,0 +1,48 @@
+#ifndef DTK_NONAME_H
+#define DTK_NONAME_H
+
+#include "mpi.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// Opaque pointer
+typedef void DTK_Map;
+
+// Possible data layouts:
+// * Contiguous 
+//     X1 X2 ... Xn Y1 Y2 ... Yn
+// * Interleave 
+//     X1 Y1 X2 Y2 ... Xn Yn
+typedef enum data_layout { CONTIGUOUS, INTERLEAVE } DTK_DataLayout;
```

Comment:
Should we make these DTK_CONTIGUOUS and DTK_INTERLEAVE to avoid collisions or does the enum prevent this?